### ISSUE TITLE
Possessive pronouns in non possessive contexts

### ISSUE THREAD

**Issue by jheinecke** [UniversalDependencies/docs#752]
In Welsh, personal pronouns are traditionally defined in a "dependent" and "independent" series. Where as the latter is used to indicate the pronominal subject and together with prepositions, the "dependent" pronouns are used either as possessives or as objects together with verbnouns:

independent pronoun

1) _you sang_
```
1   Cenaist canu  VERB _     Tense=Past|Person=2|Number=Sing root  0 _ _
2   ti      ti    PRON indep Person=2|Number=Sing            nsubj 1 _ _
```

2) _about you_
```
1 amdanat am ADP  _     Person=2|Number=Sing case 1 _ _ 
2 ti      ti PRON indep Person=2|Number=Sing root 0 _ _

```

Dependent pronoun

3) _she is seeing him_
```
Mae  bod   VERB _        _                                root  0
hi   hi    PRON indep    _                                nsubj 1
yn   yn    PART _        _                                aux  5
ei   ei    PRON dep      Gender=Masc|Person=3|Number=Sing obj  5
weld gweld NOUN verbnoun _                                xcomp 1
```

4) _his house_
```
ei ei PRON dep Gender[psor]=Masc|Person[psor]=3|Number[psor]=Sing  2 nmod:poss _ _
dŷ tŷ NOUN _   Gender=Masc|Number=Sing                             0 root      _ _
```

The dependent pronoun as a possessive indicates Gender, Person and number of the possessor, it does not at all agree with the possessed object, hence the `[psor]` extension. 
The question I'm posing myself is whether the dependent pronoun in object position (example 3) should or should not have the `[psor]` extension. There is no possession at all (one cannot possess an action), it correspond to object pronouns like _I see **him**_ in English or _je **le** vois_ in French. For the sake of homogeneity, we could add  `[psor]`, but is this a valid reason? What would be best practice?
Another question would be to use `DET` for the possessive use (as it the case in other languages (German-GSD, French-GSD, Spanish-GSD, Irish-IDT).
So in short: annotate the dependent pronouns in the same way in possessive uses and in object possession or regard them as different tokens with different UPOS and features?

**Comment by ftyers** [comment]
My feeling is that they should be `DET` when modifiers and `PRON` when standalone, and that as you said the `DET` should have `[psor]` and the `PRON` should not. I think this is how it is done in the Breton treebank, and as you say in the Irish one too. I think it's reasonable to stretch the traditional analysis here towards something more homogenous.

**Comment by dan-zeman** [comment]
I am wondering whether `[psor]` is needed at all, even in the possessive context. Layered features were invented to allow multiple instances of the same feature (with different interpretations) on one word. It is not a requirement that there are always two or more layers of the feature on the word; but if the possessor's features never co-occur with agreement features, then maybe the layers are not needed in Welsh, and simple `Gender`, `Number`, and `Person` can be used. The dependent pronoun would then feel more like a genitive form of the personal pronoun.

Regardless of whether or not the layers are used, I would treat the pronoun the same way when it modifies a possessed object and when it modifies an action. The annotation would treat the following as syntactically similar constructions (it seems to me that they actually are syntactically similar):

* _his house_
* _his action_
* _his seeing_ (although your example suggests that in Welsh it would semantically correspond to _his being seen_ because _he_ is the patient rather than the actor here)

**Comment by jheinecke** [comment]
Indeed, since the verbnoun is a noun (and not an infinitive), arguments which would be objects of verbs (if the form was finite) are genitives of the verbnoun: 
* _ei dŷ_ "his house"
* _ei weld_ "his being seen"

also with nouns or proper nouns: 
* _tŷ'r brenin_ "the hous of the king"
* _tŷ Ioan_, "Ioan's house"
* _gweld y brenin_ "the seeing of the king" (i.e. "the being seen")
*  _gweld Ioan_ "the being ssen of Ioan".

However, some languages in annotate possessive pronouns as `DET`, especially if they are in the same syntactic context (including the other Celtic languages, Breton, Scottish-Gaelic, Irish and Manx). I think the Irish case is very similar to Welsh (@tlynn747 can you confirm?), yet the Irish TB uses `DET` for pronouns in possessive constructions and `PRON` in other cases.

Do I understand ocrrectly that `[psor]` is only needed if the feature is ambiguous, as for instance in German:
```
unser   unser   DET     PPOSAT  Case=Nom|Gender=Neut|Number=Sing|Number[psor]=Plur|Person=1|Poss=Yes|PronType=Prs
```
How can we verify whether there is agreement with the head (`Number`) or not (`Number[psor]`)? In Welsh we have (currently):
```
1 ein    ein    PRON    dep     Number=Plur|Person=1|Poss=Yes|PronType=Prs    nmod:poss 2
2 tŷ     tŷ     NOUN    noun    Gender=Fem|Number=Sing                        root      0
```
and we cannot say whether the possessor is Plural or the head (in this example "our house" it's the posessor, the pronouns does not agree with it's head, unlike German, French and other IE languages)

**Comment by dan-zeman** [comment]
In German, the possessive word works like adjective/determiner `DET` because it occurs before the possessed entity and its from agrees with the possessed entity:

* _unseres Kind_ "our child" (neuter singular)
* _unsere Kinder_ "our children" (plural)

We thus need `Number=Sing` resp. `Number=Plur` to reflect the agreement suffix (_-es_ vs. _-e_), and this feature is independent of the lexical plural of "our" (< "we"), hence we add `Number[psor]=Plur` to distinguish it from the agreement `Number`.

---

In Japanese, a personal pronoun is made possessive by adding the genitive suffix (or postposition), the same way as it would work with a full-nominal-possessor. The pronoun may express plurality of the possessor but the number of the possessed entity is irrelevant:

* _watashi no kodomo_ "my child/ren" (singular possessor)
* _watashitachi no kodomo_ "our child/ren" (plural possessor)
* _Hiroshi no kodomo_ "Hiroshi's child/ren" (full nominal possessor)

It is thus a `PRON` rather than a `DET`. We may use `Number=Sing/Plur` to distinguish _watashi_ from _watashitachi_ but we don't need additional layers of `Number`.

---

In Czech, some possessives have the agreement forms just as in German, and we treat them as `DET`, with the additional `[psor]` layer:

* _náš syn_ "our son"
* _naši synové_ "our sons"

Third-person possessives are indeclinable (unlike in other, e.g., South Slavic languages) and their form is similar or identical with the genitive of the personal pronoun, so one could say they are closer to what happens in Japanese:

* _jeho syn/synové_ "his son/sons"
* _jejich syn/synové_ "their son/sons"

However, the possessive pronoun still must come before the possessed noun, while full nominal in genitive would come after the modified noun. This fact and the parallelism with the first and second persons (and with related languages) leads me to still treat them as `DET` rather than `PRON`. Consequently, the possessor's gender and number is still annotated as the `[psor]` layer, although there is no agreement gender and number in this case.

---

In English, there is no agreement between the possessive pronoun and the possessed entity, hence it is possible to treat it as in Japanese, i.e., a genitive form of the personal `PRON`, attached as `nmod(:poss)`. I think that this is more or less what is done in English UD, except for marking _my_ as a `Case=Gen` form of the lemma _I_. My feeling is that English is at least historically similar to German but its agreement morphology has been lost; therefore, I would not object if the English possessive pronouns were annotated as `DET`. But `PRON` is certainly not wrong. And no `[psor]` layer is needed.

* _my child/ren_
* _our child/ren_

Unlike in Czech, in English the word order does not necessarily distinguish possessives from genitives. While many genitive constructions would be translated to English using the preposition _of_, and the modifying nominal would come after the modified nominal, there is also the saxon genitive, which would come before the modified noun, i.e., the same as with the possessive pronoun:

* _John's child/ren_

And we certainly don't want to say that _John's_ is a determiner.

---

Based on what you write above, I think that Welsh partially patterns with English in this overview. It seems that you can decide whether `PRON` or `DET` is better and you don't need `[psor]` as an extra layer.

**Comment by jheinecke** [comment]
I agree with the choice between `PRON` and `DET` (I'll stick to `PRON` for the time being, since that is the current annotation :-)

But not adding `[psor]`this leaves `Number` (and `Gender`) ambiguous. You must know the language/documentation to know whether `Number` refers to the possessor or the possessed entity. Wouldn't it be nice to have a universal usage for this?

**Comment by dan-zeman** [comment]
You must always know the language documentation to understand all details of a feature. That is not specific to possessives. (And layered features are by definition language-specific, although we obviously want to make sure that if a layer is defined for the possessor, it is `[psor]` in all languages that need it.)

So I think you can use `[psor]` in these cases and document it in Welsh but there will probably be other languages that will not do it. If the layer is used, I would still not try to determine semantically whether it is possession=ownership, or a relation between an argument and a verbal noun, as long as they behave the same way morphosyntactically.

Related: Possessive pronouns/determiners distinguish `Person` and it is lexical person of the possessor, not agreement with the possessed entity (which would be mostly the 3rd person anyway). It is normally not layered in UD treebanks (it is `Person`, not `Person[psor]`). AFAIK, `Person[psor]` is used in languages that use a different strategy of marking pronominal possession: they use bound morphemes on possessed nouns.
